polaris-react/scripts/gen-media-queries.js

```diff
@@ -59,5 +61,5 @@ function getDownMediaCondition(breakpoint) {
 }
 
 function toEm(value) {
-  return `${parseInt(value, 10) / 16}em`;
+  return `${parseInt(value, 10)}em`;
```

Comment:
I need to revert this change once https://github.com/Shopify/polaris/pull/6012 ships